### ISSUE TITLE
Should check the subclasses of class in whiteList ?

### ISSUE THREAD

**Issue by ahyin007** [Zepo/MLeaksFinder#38]
Since we may override a class !

**Comment by Zepo** [comment]
Yes, you can do so. However, not checking the subclasses works good for me so far, cause the classes in whitelist are usually system private classes that you won't override at all.
